Documentation/specs/runtime-configuration-file.md

```diff
@@ -35,9 +35,10 @@ The files are both JSON files stored in UTF-8 encoding. Below are sample files.
 
         "framework": {
             "name": "Microsoft.DotNetCore",
-            "version": "1.0.1",
-            "rollForward": false
+            "version": "1.0.1"
         }
+        
+        "applyPatches": false
```

Comment:
I assume this is just a documentation change and we shipped the host with that parameter name correct? Otherwise it will be a breaking change.